packages/ui/src/components/ValidatorList/KeystoresDataGrid.tsx

```diff
@@ -84,6 +192,36 @@ export default function KeystoresDataGrid({
       headerClassName: "tableHeader",
       width: 360,
     },
+     // Only render Smooth column if mevSpFeeRecipient is not null (mainnet or prater)
+    ...(mevSpFeeRecipient != null && (network === "mainnet" || network ==="prater") ? [{
+      field: "spSubscription",
+      headerName: "Smooth",
+      description:
+        "Dappnode's Smooth subscription status. Smooth states can take up to 40 minutes to update.",
+      disableReorder: true,
+      disableColumnMenu: true,
+      disableExport: true,
+      sortable: false,
+      align: "center"  as GridAlignment,
+      headerAlign: "center"  as GridAlignment,
+      headerClassName: "tableHeader",
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      renderCell: (rowData: { row: any; }) => {
+        // only render smooth status if tag is "solo"
+        if (rowData.row.tag === "solo") {
```

Comment:
Needs to be checked, but probably you can set Smooth for any tag that has an editable fee recipient, not only solo

Reply:
for now, lets keep oracle status only to "solo" tag validators